be/src/http/action/meta_action.cpp

```diff
@@ -63,6 +63,8 @@ Status MetaAction::_handle_header(HttpRequest* req, std::string* json_meta) {
     tablet->generate_tablet_meta_copy(tablet_meta);
     json2pb::Pb2JsonOptions json_options;
     json_options.pretty_json = true;
+    // default value of bytes_to_base64 is true, compatible with previous config
+    json_options.bytes_to_base64 = config::enable_byte_to_base64;
```

Comment:
How about make it as a uri parameter, it would be more flexible?

Reply:
good idea!